md-dao/src/main/java/com/wipro/ats/bdre/md/dao/ProcessDAO.java

```diff
@@ -177,6 +177,28 @@ public void delete(Integer id) {
             Process process = (Process) session.get(Process.class, id);
             process.setDeleteFlag(true);
             session.update(process);
+            List<Process> subProcessList = subProcesslist(id);
+            if (subProcessList.size() > 0){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1155)